### DISCUSSION TITLE
adjust acrylic titlebar color?

### DISCUSSION THREAD

**Discussion by Neuroburst** [MicaForEveryone/MicaForEveryone#175] [Ideas]
hi I noticed the acrylic titlebars looked a bit too light even in dark mode, so is it possible to adjust the darkness manually and maybe an option for a custom titlebar color perhaps?

(this is my first discussion, so please excuse if i did something wrong)

Mica:
![Screenshot 2022-11-10 172452](https://user-images.githubusercontent.com/69093127/201240650-fff6094d-5677-499f-a135-3c803cbc988b.png)

Acrylic:
![Screenshot 2022-11-10 172434](https://user-images.githubusercontent.com/69093127/201240674-c652bc98-fcf7-482d-8585-c8952e88a8c9.png)

Doesn't Acrylic seem a lot lighter?

**Comment by xmine64** [comment]
From README:
> Mica For Everyone is not responsible for rendering the effects you set, it just asks Windows to do that for you.

So no, Windows is responsible for rendering the effect and it doesn't let you to set brightness, accent color, or anything like that. It's something that MFE can't control.

**Reply by Neuroburst** [reply]
oh
